addons/core/functions/server/fnc_replaceSwRadiosServer.sqf

```diff
@@ -0,0 +1,66 @@
+/*
+    Name: TFAR_fnc_replaceSwRadiosServer
+
+    Author(s):
+        gandjustas
+
+    Description:
+        Handles (on server) getting switching radios, handles whether a manpack must be added to the player or not.
+
+    Parameters:
+        0: OBJECT - unit to replace radios
+
+    Returns:
+        Nothing
+
+    Example:
+        _unit remoteExec ["TFAR_fnc_replaceSwRadiosServer", 2];
+*/
+params ["_unit", "_defaultRadio"];
+private _replacedRadios = [];
+
+private _replaceRadio = {
+	params ["_radio"];
```

Comment:
Change the tabs into 4 spaces please. The rest of the project is already using 4 spaces